### DISCUSSION TITLE
How to specify color/texture/pattern for each gamut in visualizing multiple colour spaces gamut?

### DISCUSSION THREAD

**Discussion by hminle** [colour-science/colour#818] [Q&A] [answered]
Hi Colour team,

When I try to visualize multiple colour_spaces gamut, is there a way to specify color/texture for each gamut so that we can see them separately? Like Color Sync in MacOS
Here is what I try:
![image](https://user-images.githubusercontent.com/16201681/120369141-dcee4e00-c2e0-11eb-827a-8ac014ca7e24.png)

Here is an example from Color Sync in MacOS
ProPhoto RGB: White Volume
sRGB: Color Volume
Reference color space: Lab
![image](https://user-images.githubusercontent.com/16201681/120369230-f8f1ef80-c2e0-11eb-8170-b3f799482121.png)

Thanks.

**Comment by thomasmansencal** [comment, marked as answer]
Hi @hminle,

Please take a look at the `colour/examples/plotting/examples_volume_plots.py` example module, it shows a few ways on how to customize the figure. Keep in mind that Matplotlib is very limited when it comes to 3D plotting.

![image](https://user-images.githubusercontent.com/99779/120920497-386e7080-c713-11eb-9e63-4afaf0aca89c.png)

Cheers,

Thomas

**Reply by hminle** [reply]
Great, thank you very much @KelSolaar
